cohort-engine/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,5 @@
+eclipse.preferences.version=1
```

Comment:
slight preference to add eclipse files to gitignore

Reply:
I had been ignoring these files manually for a long time and then I finally gave in and committed them. I don't care either way. Would be happy to .gitignore if that is the group preference.

Reply:
Ideally, all IDE specific files would be git ignored.  Our IDEs should be able to generate them from the `pom.xml` file no problem, and it prevents extra noise in further PRs as the IDEs change those metadata files.
My usual `.gitignore` entries for Eclipse is as follows.
```
bin
.classpath
.project
.settings
```

Reply:
Added these to .gitignore and removed the .settings files.